misk-feature/src/main/kotlin/misk/feature/RealAppScopedDynamicConfigResolver.kt

```diff
@@ -0,0 +1,31 @@
+package misk.feature
+
+import misk.logging.getLogger
+
+internal class RealAppScopedDynamicConfigResolver<T : ValidatableConfig<T>> constructor(
```

Comment:
i'm having a hard time parsing what all the words mean in this class name. what is meant by app scoped? aren't all configs apps scoped. seems like `DynamicConfigResolver` is a pretty name IMO.

Reply:
Yeah, this is poorly based on an implementation detail (the underlying feature's name being prefixed with the app name) that isn't relevant to the abstraction. Will rename to your much better recommendation